docs/io/ascii/fast_ascii_io.rst

```diff
@@ -84,6 +86,15 @@ IPython Notebook and so enabling multiprocessing in this context is discouraged.
 Setting ``use_fast_converter`` to be ``True`` enables a faster but
 slightly imprecise conversion method for floating-point values, as described below.
 
+The ``exponent_style`` parameter allows to define a different character
+from the default ``'e'`` for exponential formats in the input file.
```

Comment:
I think you should cut the explanation below and just make a cross-reference here and say:
```
For details see the section on :ref:`fortran_style_exponents`.
```